crates/python/src/qvm/api.rs

```diff
@@ -0,0 +1,49 @@
+use std::borrow::Cow;
+use std::collections::HashMap;
+
+use super::{PyQvmResultData, RustQvmError};
+use crate::{py_sync::py_function_sync_async, qpu::client::PyQcsClient};
+
+use rigetti_pyo3::{
+    create_init_submodule,
+    pyo3::{pyfunction, PyResult},
+    ToPythonError,
+};
+
+create_init_submodule! {
+    funcs: [
+        py_get_version_info,
+        py_get_version_info_async,
+        py_run,
+        py_run_async
+    ],
+}
+
+py_function_sync_async! {
+    #[pyfunction(config = "None")]
```

Comment:
My guess is that this is automatic for `Optional<T>` types, but seems like a typo
```suggestion
    #[pyfunction(client = "None")]
```